databricks_cli/stack/api.py

```diff
@@ -58,8 +62,9 @@
 class StackApi(object):
     def __init__(self, api_client):
         self.jobs_client = JobsApi(api_client)
+        self.workspace_client = WorkspaceApi(api_client)
 
-    def deploy(self, config_path):  # overwrite to be added
+    def deploy(self, config_path, overwrite=False):  # overwrite to be added
```

Comment:
The meaning of `overwrite` (maybe rename to `overwriteNotebooks`?) is not very clear, because now we have jobs and notebooks, and will have more.
It is okey for now, could you add some TODO here to solve this issue?